Framework/Kernel/src/InternetHelper.cpp

```diff
@@ -349,7 +355,7 @@ int InternetHelper::processErrorStates(const Poco::Net::HTTPResponse &res,
   } else if ((retStatus == HTTP_FORBIDDEN) && (rateLimitRemaining == 0)) {
     throw Exception::InternetError(
         "The Github API rate limit has been reached, try again after " +
-            rateLimitReset.toSimpleString(),
+            rateLimitReset.toSimpleString() + " GMT",
         retStatus);
```

Comment:
This feels a bit out of place now that there is a GithubApiHelper derived class. As this method is already virtual GithubApiHelper could override it and take the code that checks the rate limit and produces this GitHub specific error message.